### PR TITLE
Update to node16

### PR DESCRIPTION
**Pull request by yuliya-zhukova**

### CODE REVIEW

**Comment by zhukov-maxim** [comment]
Please remove `.DS_Store` file and add it to`.gitignore`.

File: package.json [hunk 1]
```diff
@@ -1,25 +1,23 @@
 {
   "name": "german-license-plate-codes",
-  "homepage": "https://yuliya-zhukova.github.io/german-license-plate-codes",
```
Comment (zhukov-maxim) [inline package.json:3]: Please restore `homepage` property.

File: package.json [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
-    "german-license-plate-prefixes": "^2.17.0",
-    "gh-pages": "^3.1.0",
-    "react": "^17.0.1",
-    "react-bootstrap": "^1.5.2",
-    "react-dom": "^17.0.1",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^12.20.16",
+    "@types/react": "^17.0.14",
+    "@types/react-dom": "^17.0.9",
+    "bootstrap": "^5.0.2",
```
Comment (zhukov-maxim) [inline package.json:13]: Please check manually that the upgrade from Bootstrap 4 to Bootstrap 5 didn't break anything on Windows/Mac/iPhone.

File: package.json [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
-    "@types/jest": "^26.0.20",
-    "@types/node": "^12.20.5",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.2",
-    "bootstrap": "^4.6.0",
-    "german-license-plate-prefixes": "^2.17.0",
-    "gh-pages": "^3.1.0",
```
Comment (zhukov-maxim) [inline package.json:16]: Please double-check and confirm that `gh-pages` package is not required for GitHub actions or restore it.

File: package.json [hunk 4]
```diff
@@ -34,22 +32,9 @@
   },
   "eslintConfig": {
     "extends": [
-      "airbnb-typescript",
```
Comment (zhukov-maxim) [inline package.json:37]: Please restore ESLint config.

**Comment by zhukov-maxim** [comment]
Please remove both V1 and V2 folders completely. You can always restore their content using Git history.

**Review by yuliya-zhukova — COMMENTED** [review]
Done.

File: package.json (hunk 4 above)
Comment (yuliya-zhukova) [reply package.json]: Done.

**Review by zhukov-maxim — APPROVED** [review]

**Pull request merged**